From: Matthias Geerdsen <matthias@vorlons.info>
Date: Sat, 24 Mar 2012 22:31:12 +0100
Subject: remove hex2bin(), provided with php 5.4.0

---
 .../server/core/class.conversion.php               |   18 ------------------
 1 files changed, 0 insertions(+), 18 deletions(-)

diff --git a/php-webclient-ajax/server/core/class.conversion.php b/php-webclient-ajax/server/core/class.conversion.php
index 6a9aed2..a7344f5 100644
--- a/php-webclient-ajax/server/core/class.conversion.php
+++ b/php-webclient-ajax/server/core/class.conversion.php
@@ -62,15 +62,6 @@
 class Conversion {
     
 	/**
-	 * Convert hex-string to binary data
-	 * @param string $data the hexadecimal string
-	 */
-	function hex2bin($data)
-	{
-		return pack("H*", $data);
-	}
-
-	/**
 	* Convert from UTF-8 encoded string to Windows-1252 string
 	*
 	* Note that we try two different conversion methods:
@@ -1242,15 +1233,6 @@ class Conversion {
 }
 
 /**
-* Shortcut to Conversion::hex2bin
-* @see Conversion::hex2bin()
-*/
-
-function hex2bin($data) {
-	return Conversion::hex2bin($data);
-}
-
-/**
 * Shortcut to Conversion::utf8_to_windows1252
 * @see Conversion::utf8_to_windows1252()
 */
